runtime/jnichk/jnicheck.c

```diff
@@ -548,21 +548,26 @@ jniCheckCallA(const char* function, JNIEnv* env, jobject receiver, UDATA methodT
 void
 jniCheckArgs(const char *function, int exceptionSafe, int criticalSafe, J9JniCheckLocalRefState *refTracking, const U_32 *descriptor, JNIEnv *env, ...)
 {
-	J9JavaVM *vm = ((J9VMThread*)env)->javaVM;
-	PORT_ACCESS_FROM_JAVAVM(vm);
+	J9JavaVM *vm = NULL;
```

Comment:
This should be eliminated entirely (use the global). Also allows trace/warn to be initialized in the declaration.

Reply:
updated in [47d194e](https://github.com/eclipse-openj9/openj9/commit/47d194ea2e0f95330bdcc673ddb0949d29409dac)